community/glossary.md

```diff
@@ -0,0 +1,79 @@
+provides clear direction and confidence about completing it. By calling someone
+a champion, you are placing faith in their ability as a leader and visionary.
+You can read more about champions [here](concepts/champion.md)
+
+#### References
+
+ - [@Beanow definition on discourse](https://discourse.sourcecred.io/t/about-champions-and-heroes/291)
+ - [SourceCred champion concept](concepts/champion.md)
+
+### Cred
+
+Cred is a metric that describes how important a contribution (or contributor) was to a particular project. Using SourceCred's own documentation as an example: the pull request that added this term to the glossary might have 10 cred, while a different pull request that fixes a typo might have only one cred. That would indicate that the first pull request was ten times as important. Cred "flows" between contributions and contributors; so for example, if you write a pull request that earns cred, some of that cred will flow to you.
+
+Cred is project-specific, and is computed based on the project's own [contribution graph](). For more details, see the [deep dive guide on cred](https://discourse.sourcecred.io/t/a-gentle-introduction-to-cred/20) or the [concept guide here](concepts/cred.md).
+
+#### References
+
+ - [cred on discourse](https://discourse.sourcecred.io/t/a-gentle-introduction-to-cred/20)
+ - [SourceCred cred concept](concepts/cred.md)
+
+### Grain
+
+A big part of SourceCred’s vision is the desire to create project-specific cryptoassets, called "Grain" that represent contribution to or support of a project. These cryptoassets hold promise to make open-source projects financially sustainable which is currently an unsolved problem. You can read more about grain [here](concepts/grain.md).
+
+#### References
+
+ - [Grain on sourcecred discourse](https://discourse.sourcecred.io/t/grain-a-project-specific-cred-powered-token/147)
+ - [sourcegrain by @decentralion](https://github.com/sourcegrain/mission)
+ - [SourceCred grain concept](concepts/grain.md)
+
+
+## Community Glossary
+
+The "Community Glossary" is ideal if you want to participate actively
+in the SourceCred community. Terms that are used commonly used in commonity forums,
+issue trackers, or other discussion platforms are included here.
+
+### bikeshedding
+
+Bikeshedding means talking (or even arguing) about a small detail instead of a larger issue at hand. 
+If you imagine building a house and having a bike shed outside, it would be akin
+to angsting over the color and fun name for the bike shed instead of working on
+actually finishing the house. At the end of the day, you won't have a place to live
+because you were lost in details that didn't matter. Read more about bikeshedding [here](concepts/bikeshedding.md).
+
+#### References
+
+ - [What is bikeshedding](https://css-tricks.com/what-is-bikeshedding/)
+ - [SourceCred bikeshedding concept](concepts/bikeshedding.md)
+
+### deep then wide
```

Comment:
Not sure that deep-then-wide merits a glossary entry. It was really influential in orienting the SC strategy around mid 2019, at this point though I feel like the term doesn't come up so much in our discussions.

Reply:
@Beanow and others, what do you think?